src/expression.ts

```diff
@@ -773,8 +773,8 @@ function formatDecimal(literal: DecimalLiteral) {
   const exponent = literal.exponent
     ? 'e' + literal.exponent.toString()
     : literal.flavor
-    ? ''
-    : 'e';
+      ? ''
+      : 'e';
```

Comment:
These style changes are from an older version of the linter. Try rebasing to latest `main`.